modules/visual_script/visual_script_nodes.cpp

```diff
@@ -1888,6 +1890,7 @@ void VisualScriptMathConstant::_bind_methods() {
 	BIND_ENUM_CONSTANT(MATH_CONSTANT_PI);
 	BIND_ENUM_CONSTANT(MATH_CONSTANT_2PI);
```

Comment:
Maybe we don't need 2PI if we have TAU?

Reply:
I think we should relabel the "PIx2" to "TAU (PIx2)"?

Reply:
I kept it for backwards compatibility. Maybe that would be a place for a deprecation warning ?

Reply:
Break compatibility, we warned people that alpha 2 is not final anyway :wink:

Reply:
Bump.

Reply:
I did the changes, but I'm having trouble rewriting my branch on github. I amended the commit locally, and rebased on the master. There is still the original commit on the remote branch `Goutte:feat-support-tau` on github and I'm not sure if I can rewrite its history.

Of course I could make a new branch and a new PR, but where's the fun in that ?